cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -132,6 +132,11 @@ def self.config(env)
       },
       dashboard: {
         behaviors: [
+          {
+            path: '/favicon.ico',
```

Comment:
I noticed in the CloudFront hit/miss logs that `favicon.ico` was not cached and Will said it's possible that is intentional.  There might be some functionality that we have somewhere that depends on this not being cached by CloudFront.  Can't remember what is was ... usage tracking?  measuring whether to fallback video playback?